doc/source/index.rst

```diff
@@ -58,6 +58,7 @@ So let the show begin!
    guide_getting_started_microk8s_macbook
    guide_getting_started_minikube_macbook
    guide_getting_started_with_SIEM
+   guide_getting_started_with_k8s
```

Comment:
do we need to refactor some stuff from above (EKS, GKE etc). Seems repetitive?

Reply:
i can do that if needed